### PR TITLE
Fix failing the test of `internal/net/middleware`

### PR DESCRIPTION
**Pull request by hlts2**
Signed-off-by: hlts2 <hiroto.funakoshi.hiroto@gmail.com>

### Description:

I fixed failing the test of `internal/net/middleware`.

### Environment:

- Go Version: 1.14.3
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.11.5

### Types of changes:

- [x] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [x] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: internal/net/http/middleware/timeout_test.go [hunk 1]
```diff
@@ -112,7 +112,7 @@ func TestWrap(t *testing.T) {
 			}
 		}(),
 		func() test {
-			wantErr := errors.Errorf("faild")
+			wantErr := errors.New("faild")
```
Comment (kevindiu) [inline internal/net/http/middleware/timeout_test.go:115]: can you explain why `New()` works but `Errorf` doesn't work?

Comment (hlts2) [reply internal/net/http/middleware/timeout_test.go:115]: if argument of `Errorf` is nil, `Errorf` function returns nil.
The following is details.
https://github.com/vdaas/vald/blob/master/internal/errors/errors.go#L98

Comment (kevindiu) [reply internal/net/http/middleware/timeout_test.go:115]: ohhhh....thank you for your explanation. I missed the implementation detail.

**Pull request merged**